articles/tutorials/04_working_with_textures/index.md

```diff
@@ -0,0 +1,364 @@
+
+If you run the game now, you'll get the same result as before when we loaded the texture from file. The texture will be loaded in, without error. Before we move on to drawing the texture, let's briefly cover what is actually happening now that we are using the *content pipeline*.
+
+* When you opened the MGCB Editor, the *Content.mgcb* file in your project was loaded inside it. Then in the editor, you added the *logo.png* file. When you saved the changes, these changes were written back to the *Content.mgcb* file.
+* When you told the project to run, it has to perform a build first. One of the NuGet references in a MonoGame project is the *MonoGame.Content.Builder.Tasks* NuGet reference. This adds build tasks that will perform the following:
+  1. Compile the assets defined in the *Content.mgcb* file using the MonoGame Content Builder Tool (MGCB Tool) into *.xnb* files. This compilation process performs the pre-processing of the content.
+  2. Copy the compiled *.xnb* files to your game project's build directory
+* Used the `ContentManager` at runtime to load the compiled asset.
+
+The following image illustrates this workflow:
+
+<figure><img src="./images/content-pipeline-workflow-full.png" alt="Figure 4-6: MonoGame Content Pipeline Workflow"><figcaption><p><strong>Figure 4-6: MonoGame Content Pipeline Workflow</strong></p></figcaption></figure>
+
+While doing all of this may have seemed like more work than just loading from file, it doesn't actually add more work on you as the developer.
+
+* You add the assets using the MGCB Editor instead of just copying them to you *Content* directory.
+* The *MonoGame.Content.Build.Tasks* reference builds and copies the assets to your game project build directory instead of you having to edit the *.csproj* file for each asset.
+* You use the `ContentManager` to load the assets instead of a `FromFile` method call.
+
+And for the same amount of steps, you also get the benefits mentioned before of the assets being pre-processed to an optimized format for the platform(s) your game is targeting. Regardless of which method you use to load assets, the end result gives you a texture loaded and ready to be drawn. Let's move on now to drawing the texture.
+
+## Drawing a Texture
+
+As mentioned in [Chapter 03](../03_the_game1_file/index.md#looking-at-the-code), all draw logic should be done inside the `Draw` method. So locate the `Draw` method in the *Game1.cs* file and add the following after the `Clear` method call is made:

```

Comment:
Maybe insist on the notion that Draw should simply relay some information from a state pre-calculated in Update and nothing more. No complex logic should be there.